crates/router/src/core/user.rs

```diff
@@ -1631,7 +1631,7 @@ pub async fn begin_totp(
         }));
     }
 
-    let totp = utils::user::generate_default_totp(user_from_db.get_email(), None)?;
+    let totp = utils::user::two_fa::generate_default_totp(user_from_db.get_email(), None)?;
```

Comment:
It would be better if we call this function `two_factor_authentication`?

Reply:
It is a module and I feel it will be too big for a file name.
Opinions @racnan

Reply:
`two_factor_auth` ?